submit_direct_submission_observation.py

```diff
@@ -0,0 +1,98 @@
+#!/bin/env python
+"""
+submit_direct_submission_observation.py
+
+Submit an DIRECT submission observation.
+DIRECT submissions are observations that must take place at a fixed time and location,
+and are separate from the scheduler's scheduled observations.
+NOTE: your proposal must allow direct submissions for this to be accepted.
+"""
+import requests
+
+API_TOKEN = 'PlaceApiTokenHere'  # API token obtained from https://observe.lco.global/accounts/profile/
+PROPOSAL_ID = 'PlaceProposalHere'  # Proposal IDs may be found here: https://observe.lco.global/proposals/
+
+# The target of the observation
+target = {
+    'name': 'm83',
+    'type': 'ICRS',
+    'ra': 204.253,
+    'dec': -29.865,
+    'epoch': 2000
+}
+
+# Constraints used for scheduling the observation
```

Comment:
Maybe add a comment here that these constraints are basically a NOOP since this is bypassing the scheduler?